### DISCUSSION TITLE
Nether's Exoticism

### DISCUSSION THREAD

**Discussion by KiR111n** [DynamicTreesTeam/DynamicTrees#930] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/nethers-exoticism

1.20.1 Forge plz :)

![image](https://github.com/user-attachments/assets/a9b591d3-a0f4-445c-aa38-d990430e1b8d)

**Comment by 1337phreak** [comment]
1.21.1 NeoForge as well please <3
